### PR TITLE
Add partition pause/resume to the activity stream

### PR DESCRIPTION
**Pull request by Jmgr**
This PR adds two events to the activity stream: partition paused and partition resumed.

### CODE REVIEW

File: api.proto [hunk 1]
```diff
@@ -201,10 +203,23 @@ message DeleteStreamOp {
     string stream = 1;
 }
 
+// PausePartitionsOp represents a partition pausing operation.
+message PausePartitionsOp {
+    string         stream = 1;
```
Comment (paddybyers) [inline api.proto]: are these supposed to be aligned?

**Review by paddybyers — COMMENTED** [review]
Is there supposed to be some code where you actually publish the events?

Edit: ignore this ....

File: api.proto (hunk 1 above)
Comment (Jmgr) [reply api.proto]: Ah yes, well spotted.

**Comment by Jmgr** [comment]
Created upstream, so closing this.

**Pull request closed without merging**